build/source/netcdf/def_output.f90

```diff
@@ -100,21 +100,21 @@ subroutine def_output(nHRU,nSoil,infile,err,message)
   fstring = adjustl(fstring)
   fname = trim(infile)//'_'//trim(fstring)//'.nc'
   call ini_create(nHRU,nSoil,trim(fname),ncid(iFreq),err,cmessage)
-  if(err/=0)then; message=trim(message)//trim(cmessage); return; endif
-  print "(A,A)",'Created output file:',trim(fname)
- enddo
+  if(err/=0)then; message=trim(message)//trim(cmessage); return; end if
+  print*,'Created output file:',trim(fname)
```

Comment:
explicit format `'(A,A)'` can prevent unexpected newline in `fname` when `fname` is long.